corehq/apps/sms/api.py

```diff
@@ -368,13 +368,13 @@ def _get_backend_tag(backend=None, backend_id=None):
         return f'{backend.domain}/{backend.name}'
 
 
-def should_log_exception_for_backend(backend):
+def should_log_exception_for_backend(backend, exception):
     """
-    Only returns True if an exception hasn't been logged for the given backend
+    Only returns True if the exception hasn't been logged for the given backend
     in the last hour.
     """
     client = get_redis_client()
-    key = 'exception-logged-for-backend-%s' % backend.couch_id
+    key = f'exception-logged-for-backend-{backend.couch_id}-{hash(str(exception))}'
```

Comment:
nit: if we are editing it anyway, should we make this key smaller.
`exception-logged-{backend.couch_id}-{hash(str(exception))}`

Reply:
I don't think the length there is much of an issue since the number of keys should be quite low

Reply:
Cool. Just thought it was overly descriptive.